fastlane/lib/fastlane/actions/get_version_number.rb

```diff
@@ -39,10 +40,15 @@ def self.run(params)
         return version_number
       end
 
-      def self.get_project!(folder)
+      def self.get_project!(xcodeproj_path_or_dir)
         require 'xcodeproj'
-        project_path = Dir.glob("#{folder}/*.xcodeproj").first
-        if project_path
+        if File.extname(xcodeproj_path_or_dir) == ".xcodeproj"
+          project_path = xcodeproj_path_or_dir
+        else
+          project_path = Dir.glob("#{xcodeproj_path_or_dir}/*.xcodeproj").first
+        end
+
+        if project_path && File.exist?(project_path)
           return Xcodeproj::Project.open(project_path)
         else
           UI.user_error!("Unable to find Xcode project in folder: #{folder}")
```

Comment:
@rogerluan 

This is going to crash unless using new variable name😄 

```suggestion
          UI.user_error!("Unable to find Xcode project at #{project_path || xcodeproj_path_or_dir}")
```

Reply:
@rogerluan Do you want to update this? 😊  If not, I can totally update this!

Reply:
Oh, I forgot! 😅 Give me a few mins :)

Reply:
Done @joshdholtz :) thanks for the reminder!

Reply:
And nice catch @ainame ❤️